src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -127,7 +137,17 @@ func (c *controller) WaitAndUpdateNodesStatus(status *ControllerStatus) {
 
 		hostsInProgressMap := common.GetHostsInStatus(assistedNodesMap, []string{models.ClusterStatusInstalled}, false)
 		errNodesMap := common.GetHostsInStatus(hostsInProgressMap, []string{models.HostStatusError}, true)
+		cancelNodesMap := common.GetHostsInStatus(hostsInProgressMap, []string{models.HostStatusCancelled}, true)
```

Comment:
can't we use the same map as errors? why not to add cancelled status to https://github.com/openshift/assisted-installer/pull/270/files#diff-1da579fda3bc689940661d48dd744a46f23824270427207cfae4f8c7448eddd2R139
?

Reply:
It's possible under the assumption that cancel and error behaves the same (all hosts are moved to that status once it detected in the service)

Reply:
Can we validate it? I think it is same behavior

Reply:
Yes, I validated it and all host transit to either cancel or error. However, in the controller there is no unified behavior for cancel vs error: on error it tried to collect must-gather and on cancel it does not. This is why we need to have separate indication for cancel vs error